src/select-panel/index.tsx

```diff
@@ -215,23 +228,18 @@ const SelectPanel = () => {
             disabled={disabled}
           />
         )}
-
```

Comment:
please restore this empty line removed as well

Reply:
sorry vs code linter :)